scaleway/resource_instance_private_nic.go

```diff
@@ -161,6 +170,15 @@ func resourceScalewayInstancePrivateNICDelete(ctx context.Context, d *schema.Res
 		return diag.FromErr(err)
 	}
 
+	_, err = instanceAPI.WaitForPrivateNIC(&instance.WaitForPrivateNICRequest{
```

Comment:
Should be a wait for like function. Like the ones recently merged in k8s, function, ...